rllib/agents/sac/README.md

```diff
@@ -1 +1,8 @@
-Implementation of Soft Actor-Critic (https://arxiv.org/abs/1812.05905.pdf).
+Implementation of the Soft Actor-Critic algorithm:
```

Comment:
We should also update the Feature compat matrix in `rllib-env.rst` to change (todo) to **Yes** for SAC/discrete.

Reply:
+1

Reply:
done